docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -912,26 +916,28 @@ connector.
       - ``32MB``
     * - ``delta.max-split-size``
       - Sets the largest :ref:`prop-type-data-size` for a single read section
-        assigned to a worker after max-initial-splits have been processed. You
-        can also use the corresponding catalog session property
+        assigned to a worker after ``max-initial-splits`` have been processed.
+        You can also use the corresponding catalog session property
         ``<catalog-name>.max_split_size``.
       - ``64MB``
     * - ``delta.minimum-assigned-split-weight``
-      - A decimal value in the range (0, 1] used as a minimum for weights assigned to each split. A low value may improve performance
-        on tables with small files. A higher value may improve performance for queries with highly skewed aggregations or joins.
-      - 0.05
+      - A decimal value in the range (0, 1] used as a minimum for weights
```

Comment:
Reviewers, made no changes here, just wrapped lines to 80 char. Is (0, 1] correct?